OpenRA.Mods.Common/Traits/AutoCrusher.cs

```diff
@@ -50,20 +51,17 @@ public AutoCrusher(Actor self, AutoCrusherInfo info)
 			moveInfo = self.Info.TraitInfo<IMoveInfo>();
 			nextScanTime = self.World.SharedRandom.Next(Info.MinimumScanTimeInterval, Info.MaximumScanTimeInterval);
 			isAircraft = move is Aircraft;
+			ignoresDisguise = self.Info.HasTraitInfo<IgnoresDisguiseInfo>();
 		}
 
 		void INotifyIdle.TickIdle(Actor self)
 		{
-			if (nextScanTime-- > 0)
+			if (IsTraitDisabled || nextScanTime-- > 0)
```

Comment:
Ah and one last thing, on being enabled to should set nextScanTime to 0

Reply:
Good spot, fixed.